frontend/src/app/[locale]/(base)/organization/[id]/manage-users/legacy/page.tsx

```diff
@@ -0,0 +1,114 @@
+import { UnauthorizedError } from "src/errors";
+import withFeatureFlag from "src/services/featureFlags/withFeatureFlag";
+import { getOrganizationDetails } from "src/services/fetch/fetchers/organizationsFetcher";
+
+import { useTranslations } from "next-intl";
+import { getTranslations } from "next-intl/server";
+import { redirect } from "next/navigation";
+import { GridContainer } from "@trussworks/react-uswds";
+
+import Breadcrumbs from "src/components/Breadcrumbs";
+import { AuthenticationGate } from "src/components/user/AuthenticationGate";
+
+interface InviteLegacyUsersPageProps {
+  params: Promise<{ locale: string; id: string }>;
+}
+
+const InviteLegacyUsersErrorPage = ({
+  organizationId,
+}: {
+  organizationId: string;
+}) => {
+  const t = useTranslations();
+
+  return (
+    <GridContainer>
+      <Breadcrumbs
+        breadcrumbList={[
+          {
+            title: t("OrganizationDetail.pageTitle"),
+            path: `/organization/${organizationId}`,
+          },
+          {
+            title: t("ManageUsers.pageHeading"),
+            path: `/organization/${organizationId}/manage-users`,
+          },
+          {
+            title: t("InviteLegacyUsers.pageHeading"),
+            path: `/organization/${organizationId}/manage-users/legacy`,
+          },
+        ]}
+      />
+      <h1 className="margin-top-9">{t("InviteLegacyUsers.pageHeading")}</h1>
+      <h2>{t("InviteLegacyUsers.inviteYourTeam")}</h2>
+      <div className="margin-top-1 margin-bottom-15">
+        {t("InviteLegacyUsers.dataLoadingError")}
+      </div>
+    </GridContainer>
+  );
+};
+
+async function InviteLegacyUsersPage({ params }: InviteLegacyUsersPageProps) {
+  const { locale } = await params;
+  const t = await getTranslations({ locale });
+  const { id: organizationId } = await params;
+
+  let organizationDetails;
+  try {
+    organizationDetails = await getOrganizationDetails(organizationId);
+  } catch (error) {
+    if (error instanceof UnauthorizedError) {
```

Comment:
do we need to handle this separately? My hope is that the authentication gate would handle this